cmd/handoff-bss-metadata.go

```diff
@@ -143,17 +165,19 @@ func getKernelCommandlineArgs(ncn sls_common.GenericHardware, cmdline string) st
 			// Storage NCNs get different assets than masters/workers.
 			if extraProperties.SubRole == "Storage" {
 				path = cephPath
-				version = desiredCEPHVersion
+				version = desiredCephVersion
 			} else {
 				path = k8sPath
 				version = desiredKubernetesVersion
 			}
 
-			cmdlineParts[i] = fmt.Sprintf("metal.server=http://rgw-vip.nmn/ncn-images/%s/%s", path, version)
```

Comment:
We (Shane and I) are pretty sure that `rgw-vip.nmn` isn't needed in the path, since it doesn't exist in the paths for initrd and kernel but is added in by bss.

We're going to verify that on surtur today.

Reply:
Verified, not needed. `metal.server=s3://boot-images` works.